packages/point-and-click/src/index.tsx

```diff
@@ -171,6 +171,31 @@ const StepEditor: FC<{
   getParentBound: () => Bounding;
   onBackButtonClick?: () => void;
 }> = ({ step, setStep, onBackButtonClick, getParentBound }) => {
+  const [basedOn, setBasedOn] = useState("html");
+  const [cssSelector, setCssSelector] = useState("");
```

Comment:
There shouldn't be a need for `basedOn` and `cssSelector`, they seem like duplicate state (and I'm seeing synchronization code below). `basedOn` can be deduced by whether the `step.trigger?.selector` is `undefined` or not, and the selector should just be stored on the step directly.

Reply:
We can definitely discuss this further, my main take on it is that duplicated state is both less clear and a risk for synchronization bugs as well as excessive re-renders (I normally don't care about this but here we have a lot of global events left and right).

As for your second point, we do not have any hard evidence on what we will be doing in the future, in fact it is not certain that the selector feature will maintain user interest either. It is more about providing an escape hatch to the user if the regular click solution isn't working.